packages/embark/src/cmd/cmd_controller.js

```diff
@@ -6,7 +6,8 @@ const fs = require('../lib/core/fs.js');
 
 require('colors');
 
-let version = require('../../package.json').version;
+const PKG = require('../../package.json');
+let version = PKG.version;
```

Comment:
Actually, I'd let config.js do the package.json require and let it be the only source of truth for it